tests/customer/test_C_09_PUT_cambiar_contraseña_del_customer.py

```diff
@@ -0,0 +1,110 @@
+    response_json = send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "Demos124#", old_password="incorrecto")
+    assert_schemas(response_json, 'put_update_password_customer.json')
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC5_PUT_verificar_status_code_400_cuando_el_nuevo_password_es_demasiado_largo(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "Demos"*20, TestData.old_password)
+    assert_response_status(TestData.response_status_code, 400)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC6_PUT_verificar_status_code_401_cuando_falta_password_actual(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "Demos124#", "")
+    assert_response_status(TestData.response_status_code, 401)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC7_PUT_verificar_status_code_400_cuando_falta_password_nuevo(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "", TestData.old_password)
+    assert_response_status(TestData.response_status_code, 400)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC8_PUT_verificar_status_code_400_cuando_password_tiene_caracteres_invalidos(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "demo/?<>#", TestData.old_password)
+    assert_response_status(TestData.response_status_code, 400)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC9_PUT_verificar_status_code_400_cuando_password_tiene_solo_numeros(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "12345678", TestData.old_password)
+    assert_response_status(TestData.response_status_code, 400)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC10_PUT_verificar_status_code_400_cuando_password_no_contiene_numeros(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "Demos", TestData.old_password)
+    assert_response_status(TestData.response_status_code, 400)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC11_PUT_verificar_status_code_400_cuando_password_tiene_menos_de_8_caracteres(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"], "De", TestData.old_password)
+    assert_response_status(TestData.response_status_code, 400)
+
+
+@pytest.mark.smoke
+@pytest.mark.functional
+@pytest.mark.regression
+def test_C9TC12_PUT_Verificar_status_code_401_cuando_el_id_del_customer_es_incorrecto(setup_module):
+    send_request_of_update_password_of_a_customer(TestData.module_response_json["id"]+1, "Demos124#", TestData.old_password)
```

Comment:
replace `TestData.module_response_json["id"]+1` to `99999`

Reply:
modificaso segun sugerencia realizada